pkg/kn/commands/revision_get.go

```diff
@@ -65,3 +76,38 @@ func NewRevisionGetCommand(p *KnParams) *cobra.Command {
 	revisionGetFlags.AddFlags(revisionGetCommand)
 	return revisionGetCommand
 }
+
+func getRevisionInfo(args []string, namespace string, client serving.ServingV1alpha1Interface, cmd *cobra.Command) (interface{}, error) {
+
+	switch len(args) {
+	case 0:
+		revision, err := client.Revisions(namespace).List(v1.ListOptions{})
+		if err != nil {
+			return nil, err
+		}
+		if len(revision.Items) == 0 {
+			fmt.Fprintf(cmd.OutOrStdout(), "No resources found.\n")
+			return nil, nil
+		}
+		revision.GetObjectKind().SetGroupVersionKind(schema.GroupVersionKind{
+			Group:   "knative.dev",
```

Comment:
The coordinates are wrong (whould be `serving.knative.dev` for the  Group), but we should get #134 merged as soon as possible which avoids this kind of hardcoding of the GKV. @sixolet @cppforlife can we please get #134 merged soon to avoid piling up on this technical debt ?)

Reply:
Sure...Once that is merged i will rechange this